src/Algorithms/Abstraction.php

```diff
@@ -204,7 +202,6 @@ protected function shortestDistanceToSegment(
         list( $ax, $ay ) = $a->getCoordinates();
         list( $bx, $by ) = $b->getCoordinates();
         list( $cx, $cy ) = $c->getCoordinates();
-        $return = 0.0;
```

Comment:
This is not an unused variable, but a prototype to hint at debuggers that `$return` is & always will be a double.